packages/wallets/src/features/accounts/modules/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Button, useDevice } from '@deriv-com/ui';
+import './Footer.scss';
+
+type TFooterProps = {
+    backText?: string;
+    disableBack?: boolean;
+    disableNext?: boolean;
+    nextText?: string;
+    onClickBack?: VoidFunction;
+    onClickNext?: VoidFunction;
+};
+
+const Footer: React.FC<TFooterProps> = ({
+    backText = 'Back',
+    disableBack = false,
+    disableNext = false,
+    nextText = 'Next',
+    onClickBack,
+    onClickNext,
+}) => {
+    const { isMobile } = useDevice();
```

Comment:
It's better to use `!isDesktop` instead of `isMobile` because it targets other screens other than desktop screens i.e. tablet, mobile.